packages/webapp/src/routes/developer/development-project-information.js

```diff
@@ -0,0 +1,113 @@
+import developerConstants from '../../utils/developer-constants.js'
```

Comment:
Use `constants.js` rather than `developer-constants.js`

And change below to use `constants.` rather than `developerConstants.`

Reply:
Done